src/main/java/net/fabricmc/loader/api/entrypoint/EntrypointContainer.java

```diff
@@ -34,4 +34,9 @@ public interface EntrypointContainer<T> {
 	 * Returns the mod that provided this entrypoint.
 	 */
 	ModContainer getProvider();
+
+	/**
+	 * Returns a string representation of the entrypoint.
+	 */
+	String getEntrypointName();
```

Comment:
This is a breaking change and the method should have a default impl to avoid that (it could return the mod id for example)

Reply:
Added b9e4ef1 to resolve this